demos/dev_mode_key_provisioning/src/aws_key_provisioning_demo.c

```diff
@@ -0,0 +1,675 @@
+/* Standard includes. */
```

Comment:
I don't see the modified solution/project files as part of this PR. Can you please include them?

Reply:
I had given direction to Sukhmani to add that after her demo today. We acknowledge those files need to be added, but with the timing of her demo and presentation this afternoon, she will handle that change later today.

Reply:
All project files should be updated now :) Thanks!